app/javascript/elm/src/TimeRange.elm

```diff
@@ -0,0 +1,56 @@
+module TimeRange exposing (TimeRange(..), defaultTimeRange, update, viewTimeFilter)
+
+import Html exposing (Html, div, h4, input, text)
+import Html.Attributes as Attr
+import Json.Decode as Decode
+import Json.Encode as Encode
+
+
+type TimeRange
+    = TimeRange Range
+
+
+type alias Range =
+    { timeFrom : Int
+    , timeTo : Int
+    }
+
+
+defaultTimeRange : TimeRange
+defaultTimeRange =
+    TimeRange
+        { timeFrom = 1
+        , timeTo = 1
+        }
+
+
+update : TimeRange -> Encode.Value -> TimeRange
+update timeRange newValue =
+    let
+        result =
+            Decode.decodeValue timeRangeDecoder newValue
+    in
+    case result of
+        Ok decodedValue ->
+            TimeRange
+                { timeFrom = decodedValue.timeFrom
+                , timeTo = decodedValue.timeTo
```

Comment:
I think in this module `epochFrom` and `epochTo` could be a good option for the naming

Reply:
I'm not sure why? Epoch just means the date (https://en.wikipedia.org/wiki/Epoch_(computing)) so epochFrom is a bit confusing. Did you mean something like `unixTimeForm`, `unixTimeTo`?

Reply:
Something like that 👍